README.md

```diff
@@ -63,14 +63,27 @@ This will automatically wire-up `sinon`'s setup & restoration to QUnit's `testSt
 In each test you are able to access `sinon` via the `sinon` object available as an import in your tests:
 
 ```js
+import { module } from 'qunit';
+// Use test from ember-qunit directly instead of ember-sinon-qunit
```

Comment:
I would prefer to keep references to the old API out of the primary documentation, because it will introduce confusion for users who have never used the old versions of this package.